docs/source/api/user-guide/combinators/index.rst

```diff
@@ -27,37 +27,45 @@ In this case, it is only necessary for the types to conform.
 
 In the following example we chain :py:func:`opendp.measurements.make_base_discrete_laplace` with :py:func:`opendp.transformations.make_sum`.
 
-.. doctest::
-
-    >>> import opendp.prelude as dp
-    >>> dp.enable_features("contrib", "floating-point")
-    ...
-    >>> # call a constructor to produce a transformation
-    >>> sum_trans = dp.t.make_sum(
-    ...     dp.vector_domain(dp.atom_domain(bounds=(0, 1))), 
-    ...     dp.symmetric_distance()
-    ... )
-    >>> # call a constructor to produce a measurement
-    >>> lap_meas = dp.m.make_laplace(sum_trans.output_domain, sum_trans.output_metric, scale=1.0)
-    >>> noisy_sum = dp.c.make_chain_mt(lap_meas, sum_trans)
-    ...
-    >>> # investigate the privacy relation
-    >>> symmetric_distance = 1
-    >>> epsilon = 1.0
-    >>> assert noisy_sum.check(d_in=symmetric_distance, d_out=epsilon)
-    ...
-    >>> # invoke the chained measurement's function
-    >>> dataset = [0, 0, 1, 1, 0, 1, 1, 1]
-    >>> release = noisy_sum(dataset)
+.. tabs::
```

Comment:
For me, these tabs don't render well on the dark theme.

Reply:
I hadn't checked that -- Good catch.